.bcr/presubmit.yml

```diff
@@ -38,6 +38,7 @@ tasks:
     build_flags:
     - '--cxxopt=/std:c++17'
     - '--host_cxxopt=/std:c++17'
+    - '--define=protobuf_allow_msvc=true'
```

Comment:
Should we consider adding this to https://github.com/grpc/grpc/blob/master/tools/bazel.rc?

Reply:
gRPC's bazel.rc won't be used when BCR runs the presubmit test.